cmake_modules/go_gold.sh

```diff
@@ -93,4 +93,9 @@ if [ -e helm/hpcc/Chart.yaml ] ; then
   doit "git commit -a -s -m \"$HPCC_NAME Helm Charts $HPCC_SHORT_TAG Release Candidate $HPCC_SEQUENCE\""
   doit "git push $REMOTE master $FORCE"
   popd
-fi
\ No newline at end of file
+fi
+
+# upmerge back to minor closedown branch to ensure incr.sh works
+doit "git checkout candidate_$HPCC_MAJOR.$NEW_MINOR.x"
+doit "git merge --no-ff --no-commit -s ours candidate_$HPCC_MAJOR.$NEW_MINOR.$NEW_POINT"
```

Comment:
Should this be -s recursive -X ours?

Reply:
Why?

The expectation is there is nothing to merge, other than version.cmake (which we don't want to merge).